tests/acceptance/features/lib/UsersPage.php

```diff
@@ -823,6 +826,14 @@ public function addOrRemoveUserToGroup(Session $session, $user, $group, $add=tru
 			$groupLabel->click();
 			$this->waitForAjaxCallsToStartAndFinish($session);
 		}
+		$activeDropDown = $this->find('xpath', $this->activeDropDownXpath);
```

Comment:
This is required because previously this method didn't close the group select dropdown after this method's job is complete (which is adding a user to a group or removing them from the group). Now, to perform other action (such as adding another user to a group) on the webUI after user is added or removed from the group the dropdown needs to be closed.